mongorestore/mongorestore.go

```diff
@@ -34,6 +34,12 @@ const (
 	progressBarWaitTime = time.Second * 3
 )
 
+const (
+	DEPRECATED_DB_AND_COLLECTION_WARNING = "The --db and --collection flags are deprecated for " +
```

Comment:
I don't think this needs to be exported. In go, if an element starts with a capital letter, it is "exported", meaning consumers of the package defining that element can use it. In this case, there's no need for anything outside this package to use this error message (the tests are in the same package for `mongorestore`), so let's no export it.